contrib/get_mg_inccn_data.sh

```diff
@@ -16,7 +16,7 @@ BASEDIR=$MYDIR/..
 
 # Change to directory containing the physics input data, download and extract archive
```

Comment:
This file can be deleted since we are not supporting M-G anymore

Reply:
I disagree. Even though it's not supported, the MG scheme is still in the repo and any research done with this (even with the v6 release where it is not supported) will need this data.

Reply:
Ah, sorry I just saw your comments in Slack. Ignore this request.